src/schemas/v1/incentive.ts

```diff
@@ -82,12 +139,20 @@ export const API_INCENTIVE_SCHEMA = {
     },
     start_date: {
       type: 'string',
+      description: `The time period when the incentive began, or will begin, \
+to be available. Format is similar to an ISO-8601 date, but with some \
+modifications and additions. Examples:
+
+- \`2025\`: incentive begins some time during that year.
+- \`2025-03\`: incentive begins some time during that month.
+- \`2025-03-10\`: incentive begins that day.
+- \`2025H2\`: incentive begins some time during that half-year.
+- \`2025Q3\`: incentive begins some time during that quarter.`,
     },
     end_date: {
       type: 'string',
-    },
-    special_note: {
-      type: 'string',
+      description: `The date when the incentive stopped, or will stop, being \
+available. Format is the same as for \`start_date\`.`,
     },
     ami_qualification: {
       type: 'string',
```

Comment:
I haven't bothered with these four fields because I'm planning to drop them soon. (All of these concepts can be absorbed into the low-income-thresholds concept, and I think it might be better to expose low-income thresholds in the API the same way we express them in the backend, if at all.)

Reply:
Do you think we should mark these fields as deprecated then?